src/clojure/monger/util.clj

```diff
@@ -73,3 +73,8 @@
   (get-id
     [^IPersistentMap object]
     (or (:_id object) (object "_id"))))
+
+(defn into-array-list
+  "Coerce a j.u.Collection into a j.u.ArrayList."
+  ^java.util.ArrayList [^java.util.Collection coll]
```

Comment:
Again, the idea here is probably to have a return type hint but that's not how it works in Clojure (and not really necessary in many cases since there's limited type inference in the compiler).